### PR TITLE
Implement support for columns in vimcmd_file

### PR DESCRIPTION
**Pull request by fsouza**
Something I noticed while using `rg` locally with `--column`.

Let me know what you think.

### CODE REVIEW

File: lua/fzf-lua/actions.lua [hunk 1]
```diff
@@ -42,12 +42,23 @@ end
 M.vimcmd_file = function(vimcmd, selected)
   if not selected or #selected < 2 then return end
   for i = 2, #selected do
-    -- check if the file contains line
-    local file, line = selected[i]:match("^([^:]+):(%d+)")
-    if file and line then
-      vim.cmd(string.format("%s +%s %s", vimcmd, line, vim.fn.fnameescape(file)))
+    -- check if the file contains line and col or just line.
+    local file, line, col = selected[i]:match("^([^:]+):(%d+):(%d+):")
```
Comment (fsouza) [inline lua/fzf-lua/actions.lua:46]: Unfortunately Lua doesn't support optional group captures in patterns 😭

**Comment by ibhagwan** [comment]
> Unfortunately Lua doesn't support optional group captures in patterns sob

Yes :/ it feels so ugly to have to do so many `:match`es, I was going to refactor the actions anyways and send `entry { file = .., bufnr = ... }` etc so that it's easier and clearer how to create new actions.

File: lua/fzf-lua/actions.lua [hunk 2]
```diff
@@ -42,12 +42,23 @@ end
...
+      file, line = selected[i]:match("^([^:]+):(%d+):")
+      if file then
+        col = 1
+      else
+        file = selected[i]
+      end
+    end
+
+    file = vim.fn.fnameescape(file)
+    print(file, line, col)
```
Comment (fsouza) [inline lua/fzf-lua/actions.lua:57]: Ops this was me debugging. Will fix

**Comment by ibhagwan** [comment]
I implemented this more simply using already existing code in `path.entry_to_file`, if the results now contain a column it will set the cursor to the exact spot, feels right :-)

Sorry I couldn't merge this but we got the solution nonetheless, take a look here https://github.com/ibhagwan/fzf-lua/commit/2ae75cba9691df07cb4a0f1981bd21ac2402c814

**Comment by fsouza** [comment]
Not a problem, if it works it works! :)

**Pull request closed without merging**